src/explorer/AzureAccountTreeItem.ts

```diff
@@ -190,7 +190,7 @@ export class AzureAccountTreeItem extends AzExtParentTreeItem {
       subscriptionId: subscription.subscriptionId || "",
       subscriptionPath: `/subscriptions/${subscription.subscriptionId}`,
       tenantId: subscription.tenantId || "",
-      userId: session.account.id,
+      userId: session.account.label,
```

Comment:
this filed is align with azure account, debugging with azure account, this userId is user's email.